packages/orchestration/README.md

```diff
@@ -1,14 +1,14 @@
 # @sap-ai-sdk/orchestration
 
-This package incorporates generative AI orchestration capabilities into your AI activities in SAP AI Core and SAP AI Launchpad.
+This package provides generative AI orchestration capabilities, integrating seamlessly with SAP AI Core and SAP AI Launchpad to enhance your AI workflows.
 
 ### Installation
 
 ```
 $ npm install @sap-ai-sdk/orchestration
 ```
 
-## Pre-requisites
+## Prerequisites
```

Comment:
There are two prerequisite sections... One should suffice, right?

Reply:
Merged both sections now.